deployer/deployer-services/src/main/java/io/hyscale/deployer/services/util/KubernetesVolumeUtil.java

```diff
@@ -90,6 +90,10 @@ public static String getVolumeName(V1PersistentVolumeClaim pvc) {
 		}
 		String pvcName = pvc.getMetadata().getName();
 		Map<String, String> labels = pvc.getMetadata().getLabels();
```

Comment:
Why ? 
It's not compatible with version < 1.15 right !

Reply:
This will work for versions 1.15 and above, and for other versions, we still have the fallback part.